.vscode/launch.json

```diff
@@ -26,22 +26,33 @@
                 "RADAR_DEBUG_LOGGING": "true"
             },
             "sourcePaths": [
-                "build/generated/sources/annotationProcessor/java/main"
+                "**src/main/**",
+                "build/generated/sources/annotationProcessor/java/main",
+                "/home/mike/projects/javalin/**src/main/java/**"
```

Comment:
Its a little weird and not portable that /home/mike paths have crept into the repo but I haven't been using vscode so idk.  If its helpful to keep the file in the repo I guess its fine.

Reply:
Oops. That was me trying to look at something and those don't need to be there.